include/execution.hpp

```diff
@@ -1471,7 +1403,7 @@ namespace std::execution {
     } // namespace __impl
 
     struct __submit_t {
-      template <receiver _Receiver, sender_to<_Receiver> _Sender>
+      template <class _Receiver, sender_to<_Receiver> _Sender>
```

Comment:
Could be an artifact of an earlier edit in which I did away with the `receiver` concept entirely. I should bring these back, but I'll note that in this particular case it's unnecessary since `sender_to` enforces that `_Receiver` is a `receiver`.